dropwizard/README.md

```diff
@@ -0,0 +1,108 @@
+Dropwizard Armeria Bundle
+========================
+
+A bundle for delegating application traffic through [Armeria](https://line.github.io/armeria/index.html) endpoints in Dropwizard applications.
+
+Dependency Info
+---------------
+```xml
+<dependency>
+    <groupId>com.linecorp.armeria</groupId>
+    <artifactId>armeria-dropwizard</artifactId>
+    <version>0.95.0</version>
```

Comment:
Should probably move all this to the RST so that the version can be templated

Reply:
Yes, please. That would be awesome.